app/src/main/java/org/oppia/app/profile/ProfileActivityPresenter.kt

```diff
@@ -15,7 +15,7 @@ class ProfileActivityPresenter @Inject constructor(
   /** Adds [ProfileChooserFragment] to view. */
   fun handleOnCreate() {
     // TODO(#482): Ensures that an admin profile is present. Remove when there is proper admin account creation.
-    profileManagementController.addProfile("Sean", "12345", null, true, R.color.avatar_background_1, isAdmin = true)
+    profileManagementController.addProfile("Sean", "", null, true, R.color.avatar_background_1, isAdmin = true)
```

Comment:
Suggest also using named arguments for the null & true parameter here (you can use them for all of the parameters to make it really explicit).